python/api/vrfq/requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/tradeparadigm/sdks.git#egg=sdk_commons&subdirectory=sdk_commons
+git+https://github.com/tradeparadigm/sdks.git#egg=opyn&subdirectory=opyn
+git+https://github.com/tradeparadigm/sdks.git#egg=ribbon&subdirectory=ribbon
```

Comment:
shouldn't we freeze the version to avoid involuntary updates?

Reply:
it is a trade-off 😅 . Doing so we may get outdated soon.
Please see the thread above https://github.com/tradeparadigm/code-samples/pull/16#discussion_r937387916

wdyt?